src/client/scripts/game/misc/math.js

```diff
@@ -265,9 +270,12 @@ const math = (function() {
 
     function areLinesCollinear(lines) {
         let gradient
-        for (line of lines) {
-            if (!gradient) gradient = line[0]/line[1]
-            if (!isAproxEqual(line[0]/line[1], gradient)) return false;
+        for (const line of lines) {
+            console.log(line)
```

Comment:
Maybe not needed anymore at this point, but is comparing the `c` value of the lines enough to tell if they are colinear? If c's are equal => colinear.